sql/moz-fx-data-shared-prod/org_mozilla_mozregression_derived/mozregression_aggregates_v1/init.sql

```diff
@@ -0,0 +1,25 @@
+CREATE OR REPLACE TABLE
+  `moz-fx-data-shared-prod`.org_mozilla_mozregression_derived.mozregression_aggregates_v1
```

Comment:
@scholtzan Despite running this I'm not seeing the table in shared-prod and CI is still failing. Am I doing something wrong?

```bash
wlach@antwerp bigquery-etl % ./bqetl query initialize org_mozilla_mozregression_derived.mozregression_aggregates_v1           
Create destination table for sql/moz-fx-data-shared-prod/org_mozilla_mozregression_derived/mozregression_aggregates_v1/init.sql
```

Reply:
Looks like the init.sql query doesn't get executed. https://github.com/mozilla/bigquery-etl/pull/1777 should fix this.

Reply:
That fixed it, thanks! (actually the query was broken, but that's my fault)